.github/workflows/ci-build.yaml

```diff
@@ -54,7 +55,7 @@ jobs:
       - uses: actions/checkout@v3
       - name: Setup PHP SDK
         id: setup-php
-        uses: cmb69/setup-php-sdk@v0.5
+        uses: cmb69/setup-php-sdk@master
```

Comment:
Have asked if a new tag can be used or we'll continue using `@master` for now: https://github.com/cmb69/setup-php-sdk/pull/21#issuecomment-1344065465

Reply:
Will leave with `@master` for now and update later.